eslint.config.mjs

```diff
@@ -104,6 +104,7 @@ export default [
         rules: {
             'accessor-pairs': ['error', { setWithoutGet: false, getWithoutSet: false }],
             'curly': ['error', 'all'],
+            'no-unused-expressions': 'off',
```

Comment:
Same as in `engine`:

https://github.com/playcanvas/engine/blob/main/eslint.config.mjs#L58

So that expressions like `.to.be.null;` do not give any errors